docusaurus-docs/conda-store/references/configuration-options.md

```diff
@@ -210,14 +210,16 @@ setting is useful if you want to protect environments from
 modification from certain users and groups. Note: this configuration
 option is not supported on Windows.
 
-`CondaStore.serialize_builds` DEPRECATED no longer has any effect
-
 `CondaStore.post_update_environment_build_hook` is an optional configurable to
 allow for custom behavior that will run after an environment's current build changes.
 
 `CondaStore.lock_backend` is the name of the default lock plugin to use
 when locking a conda environment. By default, conda-store uses [conda-lock](https://github.com/conda/conda-lock).
 
+#### Deprecated configuration options for `conda_store_server._internal.app.CondaStore`
+
+`CondaStore.serialize_builds` no longer has any effect
+
```

Comment:
Is there a reason why this is a H4 instead of a H3 like the rest of the configuration options? Being an H4 means this does not show in the in-page TOC like the rest of the config options.

Also, line 509 has `### (deprecated) `conda_store_server.registry.ContainerRegistry`, so we have inconsistent formatting; we should stick to one throughout.